internal/consensus/reactor.go

```diff
@@ -732,6 +732,62 @@ func (conR *Reactor) gossipDataForCatchup(logger log.Logger, rs *cstypes.RoundSt
 	time.Sleep(conR.conS.config.PeerGossipSleepDuration)
 }
 
+func pickVoteToSend(
```

Comment:
So, for code clarity, don't you think that `pickVoteToSend` and `pickVotesForHeight` methods should be on the same place?

So either both here, namely before `gossipVotesRoutine` method, or after it.